src/components/Input.tsx

```diff
@@ -0,0 +1,56 @@
+import { Input } from "antd";
+import styles from "@/styles/Chat.module.css";
+import { AudioFilled, SendOutlined } from "@ant-design/icons";
+import { useState } from "react";
+
+interface InputProps {
+  sendMsg: (msg: string) => void;
+}
+
+const InputMsg = (props: InputProps) => {
+  const [audioStatus, setStatus] = useState(false);
```

Comment:
can you change the name to something more clear, something similar to isRecording